### DISCUSSION TITLE
Share actual plot farming times?

### DISCUSSION THREAD

**Discussion by musicjeremy** [backblaze-b2-samples/b2fs4chia#4] [General]
Can the devs please share some representative data for chia plots check times as well as farming times?

**Comment by ppolewicz** [comment]
This depends on a huge number of factors, for example latency between harvester and B2.

I can say for myself, but most of my quality check + full proof times are below 10s per piece. On a good day it can be better, on a bad day it can be worse. My main problem used to be full node loosing sync, but chia actually patched that, I think, because I no longer have this problem.

I suggest that you test it yourself and see if it works well in your environment.
